src/components/NewWorkspaceModal.js

```diff
@@ -175,6 +195,30 @@ const NewWorkspaceModal = Utils.withDisplayName('NewWorkspaceModal', ({
           }), billingProjects)).sort()
         })
       ])]),
+      h(IdContainer, [id => h(Fragment, [
+        h(FormLabel, { htmlFor: id }, [
+          'Bucket location',
+          h(InfoBox, { style: { marginLeft: '0.25rem' } }, [
+            'A bucket location can only be set when creating a workspace. ',
+            'Once set, it cannot be changed. ',
+            'A cloned workspace will automatically inherit the bucket location from the original workspace but this may be changed at clone time.',
+            p([
+              'By default, workflow and Cloud Environment VMs will run in the same region as the workspace bucket. ',
```

Comment:
Nit: It feels like `Cloud Environments` would suffice as `VM` is not a term we mean to use a lot on Terra UI interactive analysis sections going forward, and may not be completely intuitive in the case of Spark clusters and Kubernetes clusters (where we deploy Galaxy). Similarly for line #207 below and also for [Compute Modal line #875](https://github.com/DataBiosphere/terra-ui/pull/2642/files#diff-149d7ad595448fb2f6bedcff6dde5f4cbcccd8d5148fed8820ce88190a3bc38bR875). 
```suggestion
              'By default, workflow and Cloud Environments will run in the same region as the workspace bucket. ',
```

Reply:
Done. Thanks!